src-docs/src/views/text_diff/text_diff_example.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Link } from 'react-router';
```

Comment:
As the `react-router` just got updated, so i think `Link` should be imported from `react-router-dom` . But let's give it a try, as for this pr router is still `v3`.

Reply:
Ahh good catch, yeah i just did this based on the other usages in this branch. Perhaps a good idea for you to rebase with master to get the router updates and then you can update this addition too?

Reply:
Will surely do